cpp/teensy/lib/multilateration/multilateration.h

```diff
@@ -0,0 +1,70 @@
+#pragma once
+
+#include "arm_math.h"
+#include <Arduino.h>
+
+#ifndef HYDROPHONE_DETAILS
+#define HYDROPHONE_DETAILS
+
+const int32_t NUM_HYDROPHONES = 5;
+const int32_t NUM_DIMENSIONS = 3;
+
+const float32_t HYD_0_X_POS = -0.11;
+const float32_t HYD_0_Y_POS = 0.31;
+const float32_t HYD_0_Z_POS = 0.10;
+
+const float32_t HYD_1_X_POS = 0.11;
+const float32_t HYD_1_Y_POS = 0.31;
+const float32_t HYD_1_Z_POS = 0.10;
+
+const float32_t HYD_2_X_POS = 0.0;
+const float32_t HYD_2_Y_POS = -0.24;
+const float32_t HYD_2_Z_POS = 0.0;
+
+const float32_t HYD_3_X_POS = 0.5;
+const float32_t HYD_3_Y_POS = -0.1;
+const float32_t HYD_3_Z_POS = 0.4;
+
+const float32_t HYD_4_X_POS = 0.4;
+const float32_t HYD_4_Y_POS = 0.0;
+const float32_t HYD_4_Z_POS = -0.4;
+
+#endif
+
+#ifndef PHYSICAL_CONSTANTS
+#define PHYSICAL_CONSTANTS
+
+const float32_t SOUND_SPEED = 1480.0; //[m/s]
+
+#endif
+
+#ifndef DSP_CONSTANTS
+#define DSP_CONSTANTS
+
+const int SAMPLING_FREQ = 300000; //[Hz]
+
+#endif
+
+struct Positions {
```

Comment:
```suggestion
struct Position {
```
It only becomes plural in an array ;)